commands/publish/__tests__/publish-command.test.js

```diff
@@ -239,6 +241,24 @@ Map {
       );
       expect(otplease.getOneTimePassword).toHaveBeenLastCalledWith("Enter OTP:");
     });
+
+    it("Passes otp delay if specified.", async () => {
+      const testDir = await initFixture("normal");
+
+      const otp = "123456";
+      npmPublish.mockImplementation(() => Promise.resolve());
```

Comment:
This shouldn't be necessary? It's already [manually mocked](https://github.com/lerna/lerna/blob/master/commands/__mocks__/@lerna/npm-publish.js).